src/sql/workbench/contrib/scripting/browser/scripting.contribution.ts

```diff
@@ -150,9 +152,13 @@ MenuRegistry.appendMenuItem(MenuId.ObjectExplorerItemContext, {
 		ContextKeyExpr.and(
 			ConnectionContextKey.Provider.notEqualsTo('KUSTO'),
 			ConnectionContextKey.Provider.notEqualsTo('LOGANALYTICS'),
+			TreeNodeContextKey.SubType.notEqualsTo('LedgerDropped'),
```

Comment:
This is only for tables, right? I'd suggest adding it to the Table check below then like you did for Views (same for the similar check below)